### PR TITLE
DhanuKBEBCChanges

### PR DESCRIPTION
**Pull request by dsunkari**
KB - How do i submit EBC Request Changes..

### CODE REVIEW

**Comment by nathanyoung** [comment]
The word docs mentioned in the article aren't included as attachments.

https://github.com/CenturyLinkCloud/PublicKB#attachments

File: General/how-do-i-submit-an-executive-briefing-request.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{{{
+  "title": "How do i submit an Executive Briefing Request",
```
Comment (osbornm) [inline General/how-do-i-submit-an-executive-briefing-request.md]: capital I

File: General/how-do-i-submit-an-executive-briefing-request.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+* Completed the Executive Briefing Request Form (this is attached to the bottom of the KB)
+* EBCForm.docx, can be used for Centurylink Sales Team
+* EBCFormCustomerDirect.docx can be used for any customer to request an Executive Brief
+* Email EBC@ctl.io the completed form
+
+
+### Who can submit an Executive Briefing Request
+
+Anyone. Customers, Account Reps, Sales Execs, Senior Mgmt...etc
```
Comment (osbornm) [inline General/how-do-i-submit-an-executive-briefing-request.md]: Senior Management

File: General/how-do-i-submit-an-executive-briefing-request.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{{{
+  "title": "How do i submit an Executive Briefing Request",
+  "date": "02-10-2015",
+  "author": "Dana Bowlin",
+  "attachments": [
+  {
+    "file_name": "Attached File",
```
Comment (osbornm) [inline General/how-do-i-submit-an-executive-briefing-request.md]: Shouldn't these be names something like  `Customer Direct Form` and `EBC Form`? not both named attached File?

**Pull request merged**